src/internal_events/eventstoredb_metrics.rs

```diff
@@ -25,27 +34,21 @@ pub struct EventStoreDbStatsParsingError {
 
 impl InternalEvent for EventStoreDbStatsParsingError {
     fn emit_logs(&self) {
-        error!(message = "JSON parsing error.", error = ?self.error);
+        error!(
+            message = "JSON parsing error.",
+            error = ?self.error,
+            error_type = "parse_failed",
+            stage = "processing",
+        );
     }
 
     fn emit_metrics(&self) {
+        counter!(
+            "component_errors_total", 1,
+            "stage" => "processing",
+            "error_type" => "parse_failed",
```

Comment:
Similar to above, should this be tagged with `error`?

Reply:
Ah yeah, I missed it...